qt511.patch

```diff
@@ -0,0 +1,11 @@
+--- a/libgui/src/settings-dialog.cc
```

Comment:
I think it's helpful to include the complete hg metadata in the patch file.

Reply:
Can you do that? I just copied the patch from hgweb.

Reply:
Done in https://github.com/Alexander-Wilms/org.octave.Octave/pull/1.